boot.brs

```diff
@@ -342,20 +345,38 @@ function bootSetup()
     else
       print "syncUrl found in registry: ", syncUrl
     end if
+
+    print("Sending IP, MAC address, bundle version, and player type to AccessKit API...")
+    ' These fire and forget requests must happen the config waitMessage, or they seem to get garbage collected before firing
+    ipReq = createObject("rourltransfer")
+    ipReq.setUrl(syncUrl+"/api/mediaplayer/"+id.toStr()+"/ipAddress")
+    ipReq.asyncPostFromString("password="+password+"&ipAddress="+currentIP)
+    macReq = createObject("rourltransfer")
+    macReq.setUrl(syncUrl+"/api/mediaplayer/"+id.toStr()+"/macAddress")
+    macReq.asyncPostFromString("password="+password+"&macAddress="+macAddress)
+    if versionData <> invalid then
+      versionReq = createObject("rourltransfer")
+      versionReq.setUrl(syncUrl+"/api/mediaplayer/"+id.toStr()+"/bundleVersion")
+      versionReq.asyncPostFromString("password="+password+"&bundleVersion="+versionData.version)
+    end if
+    playerTypeReq = createObject("rourltransfer")
+    playerTypeReq.setUrl(syncUrl+"/api/mediaplayer/"+id.toStr()+"/playerType")
+    playerTypeReq.asyncPostFromString("password="+password+"&playerType=BrightSign-"+model+"-"+firmwareVersion)
```

Comment:
It's not a huge deal either way but I wonder if it's better for playerType to be strictly BrightSign, and then another optional field which is model no. Ie PlayerMake and PlayerModel or something like that. Doesn't matter so much, just a thought